.github/workflows/build.yml

```
@@ -164,6 +164,37 @@ jobs:
           pushd android 1> /dev/null
           ./gradlew clean build check test
         working-directory: example
+  android-test:
+    name: "Android: instrumented tests"
+    strategy:
+      matrix:
+        api-level: [21, 29]
+    runs-on: macos-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+    - name: Set up JDK 1.8
```

I think we need to indent this block (or just run Prettier so it can format this file):
```suggestion
      - name: Set up JDK
```